controller/internal/enforcer/applicationproxy/servicecache/servicecache_test.go

```diff
@@ -128,20 +128,17 @@ func TestServiceCache(t *testing.T) {
 			cerr := c.Add(s1, "1", "first data", true)
 			So(cerr, ShouldBeNil)
 			So(c.local, ShouldNotBeNil)
-			So(len(c.local), ShouldEqual, 3)
 			So(c.localHosts, ShouldNotBeNil)
```

Comment:
The test is not testing without this. We must read the cache and validate the entries are there in order for the test ot be complete.

Reply:
I agree, but this particular test is just testing whether the add entries succeeds or not. There are following tests which are indeed doing a cache lookup after insertions.